src/nix/repl.cc

```diff
@@ -892,10 +905,23 @@ struct CmdRepl : StoreCommand, MixEvalArgs
 
     void run(ref<Store> store) override
     {
+
         evalSettings.pureEval = false;
```

Comment:
It is interesting it seems this line doesn't work?

Reply:
it takes effect after the installables are parsed.... moving it to the constructor seems to do it

Reply:
Great!